### ISSUE TITLE
[nsfw] gelbooru.com: ads

### ISSUE THREAD

**Issue by MightyKombat** [uBlockOrigin/uAssets#20852]
### Prerequisites

- [X] This is **NOT** a [YouTube](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [Facebook](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [Twitch](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://gelbooru.com/index.php?page=account&s=home`

### Category

Site breakage

### Description

Use of ublock to block as many ads as I can on gelbooru has now caused an ugly white blank space to spawn above any image. I can't use the element zapper or picker on it as it can only be erased with ##.mainBodyPadding > div:nth-of-type(1) at most, and doing that causes the Settings menu to blank.

### Other extensions used

NoScript, all scripts allowed

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.53.0
Firefox: 120
filterset (summary):
 network: 103707
 cosmetic: 43995
 scriptlet: 19124
 html: 1112
listset (total-discarded, last-updated):
 added:
  https://gitlab.com/Sorrow-San/9anime-adblock-filters/raw/master/9Anime%20filter%20list: 148-3, 2m
  https://raw.github.com/reek/anti-adblock-killer/master/anti-adblock-killer-filters.txt: 0-0, 2m
 default:
  user-filters: 100-16, never
  ublock-filters: 36600-144, 2m
  ublock-badware: 7482-10, 2m
  ublock-privacy: 835-4, 2m
  ublock-unbreak: 2197-1, 2m
  ublock-quick-fixes: 204-2, 2m
  easylist: 75664-717, 2m
  easyprivacy: 32965-738, 2m
  urlhaus-1: 9815-0, 2m
  plowe-0: 3718-7, 2m
filterset (user): [array of 102 redacted]
trustedset:
 added: [array of 24 redacted]
 removed:
  about-scheme
userSettings:
 advancedUserEnabled: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 210380 ms (selfie)
 maxAssetCacheWait: 2228 ms
popupPanel:
 blocked: 0
 extended:
  ##.footerAd2
  ##.mainBodyPadding > div:nth-of-type(1)
```
</details>

**Comment by mapx-** [comment]
> Use of ublock to block as many ads as I can on gelbooru has now caused an ugly white blank space to spawn above any image.
> NoScript, all scripts allowed

- meaning you broke the page using doubtful filters

Reset uBO to default filter lists, settings, disable all other extensions during the tests, remove ALL your custom filters (from "my filters") for gelbooru site

**Comment by Bun01294** [comment]
@mapx- 
ads are leaking again on the website
i have ubo to default, other extensions are disabled and the custom filters are gone, still getting them, 
Links:
https://gelbooru.com/index.php?page=post&s=list&tags=all [NSFW]
https://gelbooru.com/index.php?page=post&s=view&id=12901808 [NSFW]

in pc i only see them for nsfw posts, on moblie they show up both, on the middle of search results and nsfw posts. i tested them on sfw and they don't show up



uBlock Origin: 1.67.0
Firefox: 144
filterset (summary):
 network: 185238
 cosmetic: 74273
 scriptlet: 24668
 html: 2534
listset (total-discarded, last-updated):
 added:
  https://easylist-downloads.adblockplus.org/advblock.txt: 35219-0, 12h.44m
  https://easylist-downloads.adblockplus.org/cntblock.txt: 1850-4, 12h.44m
  https://filters.adtidy.org/extension/ublock/filters/1.txt: 22267-2346, 2d.4h.23m
 default:
  user-filters: 291-29, never
  ublock-filters: 41712-95, 2h.9m Δ
  ublock-badware: 10713-31, 2h.9m Δ
  ublock-privacy: 3540-9, 2h.9m Δ
  ublock-unbreak: 2760-6, 2h.9m Δ
  ublock-quick-fixes: 378-18, 2h.9m Δ
  easylist: 82814-324, 2h.9m Δ
  easyprivacy: 54759-352, 2h.9m Δ
  urlhaus-1: 31260-5, 9m
  plowe-0: 3451-948, 4d.14h.8m
filterset (user): [array of 301 redacted]
userSettings:
 userFiltersTrusted: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 151 ms (selfie)
 maxAssetCacheWait: 66 ms
 cacheBackend: indexedDB

**Comment by stephenhawk8054** [comment]
Fixed here: https://github.com/easylist/easylist/commit/9c5c52a3b1b4150df9f188a2fe7a586dd94d8350

**Comment by Bun01294** [comment]
> Fixed here: [easylist/easylist@9c5c52a](https://github.com/easylist/easylist/commit/9c5c52a3b1b4150df9f188a2fe7a586dd94d8350)

Nice, solution is working on moblie and pc.
